### DISCUSSION TITLE
Add hook and function to unregister section

### DISCUSSION THREAD

**Discussion by andreas83** [elementor/elementor#24536] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Sometimes we need to deactivate certain areas for novice users. 

### Describe the solution you'd like

Similar to the [Unregistering Controls example ](https://developers.elementor.com/docs/managers/unregistering-controls/) it would be nice to have a option to unregister sections

### Describe alternatives you've considered

Alternatively it would be very cool to have a editor, where you can sort, categorize controls and assign to specific user roles.

**Comment by Thelastpoet** [comment]
Does unregister_controls ever work with advanced tab controls that get set by default?
